### DISCUSSION TITLE
Capstone kurs IDR and USD

### DISCUSSION THREAD

**Discussion by sabrinaa311** [teamalgoritma/community#181] [Q&A] [answered]
Ketiga tag html menggunakan <td> dan <tr> tanpa class bagaimana melakukan looping `for` nya ya?
![p](https://user-images.githubusercontent.com/107026622/177960361-a2951396-22c7-4997-811a-9e9e3b89d6f1.png)

**Comment by triarm** [comment, marked as answer]
Hi @sabrinaa311 ,
Untuk mengambil bagian HTML tanpa class, Anda tidak perlu menuliskan parameter `attrs` pada methods `find()` maupun `find_all()`. Misalkan di sini Anda ingin mengambil seluruh elemen `td` maka Anda hanya perlu menuliskan:
```
table.find_all("td")
```

Semoga menjawab.
